lib/geminabox/server.rb

```diff
@@ -29,6 +29,10 @@ def allow_upload?
         Geminabox.allow_upload
       end
 
+      def base_path?
+        Geminabox.base_path.sub(%r{/\/$/}, '')
```

Comment:
> lib/geminabox/server.rb:33:37: C: [Correctable] Style/RedundantRegexpEscape: Redundant escape inside regexp literal
        Geminabox.base_path.sub(%r{/\/$/}, '')
                                    ^^

@tnir Any suggestions? I am not a ruby regex professional.

Reply:
It should be fixed now.
```diff
diff --git a/lib/geminabox/server.rb b/lib/geminabox/server.rb
index 7eecc2a..4cc8a5b 100644
--- a/lib/geminabox/server.rb
+++ b/lib/geminabox/server.rb
@@ -30,7 +30,7 @@ module Geminabox
       end

       def base_path?
-        Geminabox.base_path.sub(%r{/\/$/}, '')
+        Geminabox.base_path.sub(%r{/$}, '')
       end

       def fixup_bundler_rubygems!
```